recover-id-object/scripts/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG build_image=rust:1.70.0
+ARG base_image=debian:buster-slim
```

Comment:
The default rust image should be explicit about using a buster based image to make sure dependencies are compatible.